h2o-docs/src/product/data-science/rulefit.rst

```diff
@@ -0,0 +1,125 @@
+	- If the distribution is ``poisson``, the response column must be numeric.
+	- If the distribution is ``tweedie``, the response column must be numeric.
+	- If the distribution is ``gaussian``, the response column must be numberic.
+	- If the distribution is ``gamma``, the response column must be numeric.
+	- If the distribution is ``fractionalbinomial``, the response column must be numeric between 0 and 1.
+	- If the distribution is ``negativebinomial``, the response column must be numeric and non-negative.
+	- If the distribution is ``ordinal``, the response column must be categorical with at least 3 levels. 
+	- If the distribution is ``AUTO``,
+
+		- and the response is **Enum** with cardinality = 2, then the family is automatically determined as **bernoulli**.
+		- and the response is **Enum** with cardinality > 2, then the family is automatically determined as **multinomial**.
+		- and the response is numeric (**Real** or **Int**), then the family is automatically determined as **gaussian**.
+
+Interpreting a RuleFit Model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The output for the RuleFit model includes:
+
+- model parameters
+- rule importances in tabular form
+- training and validation metrics of the underlying linear model
+
+Examples
+~~~~~~~~
+
+.. tabs::
+	.. code-tab:: r R
+
+		library(h2o)
+		h2o.init()
+
+		# Import the titanic dataset:
+		f <- "https://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/titanic.csv"
+		titanic <- h2o.importFile(f)
+
+		# Set the predictors and response; set the factors:
+		response = "survived"
+		predictors <- c("age", "sibsp", "parch", "fare", "sex", "pclass")
+		titanic[,response] <- as.factor(titanic[,response])
+		titanic[,"pclass"] <- as.factor(titanic[,"pclass"])
+
+		# Build and train the model:
+		rf_h2o = h2o.rulefit(y = response, x = predictions, training_frame = titanic, max_rule_length = 10, max_num_rules = 100, seed = 1234)
+
+		# Retrieve the rule importance:
+		print(rf_h2o@model$rule_importance)
+
+
+	.. code-tab:: python
+
+		import h2o
+		h2o.init()
+		from h2o.estimators import H2ORuleFitEstimators
+
+		# Import the titanic dataset and set the column types:
+		df = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/titanic.csv", 
+				     col_types={'pclass': "enum", 'survived': "enum"})
+
+		# Set the predictors and response:
+		x = ["age", "sibsp", "parch", "fare", "sex", "pclass"]
+		y = "survived"
+
+		# Build and train the model:
+		rf_h2o = H2ORuleFitEstimator(max_rule_length=10, 
+					     max_num_rules=100, 
+					     seed = 1234, 
+					     model_type="rules_and_linear")
+		rf_h2o.train(training_frame=df, x=x, y=y)
+
+		# Retrieve the rule importance:
+		print(rf_h2o._model_json['output']['rule_importance'])
+
+
+References
+~~~~~~~~~~
+
+FRIEDMAN, J. H., & POPESCU, B. E. (2008). Predictive learning via rule ensembles. The Annals of Applied Statistics, 2(3), 916-954.
```

Comment:
```suggestion
`Friedman, J. H., & Popescu, B. E. (2008). Predictive learning via rule ensembles. The Annals of Applied Statistics, 2(3), 916-954.  <https://arxiv.org/abs/0811.1679>`__
```

Reply:
Not sure why I can't commit this change (adding a hyperlink) directly... Can you try @hannah-tillman ?